test/Readout.spec.js

```diff
@@ -55,7 +47,7 @@ describe('The readout', function() {
             const element = await page.$(SEL_FEEDBACK);
             const text = await page.evaluate(element => element.innerText, element);
 
-            expect(text).to.include(TEXT);
+            expect(text).to.include("Some willing", "text");
```

Comment:
Should these strings be stored in global constants?

Reply:
The reason we did this was because when we evaluate, it splits the text into 2 strings, "Some willing" and "text". This was what we found to be the most elegant solution as we needed to make sure that the strings were included, but this was the only text case where they were split. We could assign "Some willing" and "text" to global variables, but since this is a special case, that seems unnecessary.

Reply:
Okay!